docs/api/index.rst

```diff
@@ -4,5 +4,4 @@ API Documentation
 .. toctree::
    :maxdepth: 2
 
-   notes/index
```

Comment:
Can we remove "Package Reference" button and directly show `gluonnlp.xx` under API Documentation? We can make it flat. 
![Screen Shot 2020-02-18 at 10 22 22 AM](https://user-images.githubusercontent.com/5545640/74765510-946c6780-5238-11ea-993d-5533d0f00d23.png)